### ISSUE TITLE
Focal Loss

### ISSUE THREAD

**Issue by PJHardy** [cfzd/Ultra-Fast-Lane-Detection#126]
Hi, 

I found SoftmaxFocalLoss does not have the parameter "alpha",  how to add it in this function?

```
class SoftmaxFocalLoss(nn.Module):
    def __init__(self, gamma, ignore_lb=255, *args, **kwargs):
        super(SoftmaxFocalLoss, self).__init__()
        self.gamma = gamma
        self.nll = nn.NLLLoss(ignore_index=ignore_lb)

    def forward(self, logits, labels):
        scores = F.softmax(logits, dim=1)
        factor = torch.pow(1.-scores, self.gamma)
        log_score = F.log_softmax(logits, dim=1)
        log_score = factor * log_score
        loss = self.nll(log_score, labels)
        return loss
```

**Comment by cfzd** [comment]
@PJHardy 
You can use the implementation in here:
https://github.com/clcarwin/focal_loss_pytorch/blob/master/focalloss.py
